keyboards/durgod/k3x0/keymaps/chimera/keymap.c

```diff
@@ -0,0 +1,373 @@
+/* Copyright 2021 Don Kjer and Tyler Tidman
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+// Layer shorthand
+enum _layer {
+    _WBL,
+    _WFL,
+    _WSL,
+    _MBL,
+    _MFL,
+    _MSL
+};
+
+#ifndef DEFAULT_LAYOUT
+# define DEFAULT_LAYOUT _WBL // Change to _MBL to boot into Mac layout.
+#endif
+#ifndef ALT_LAYOUT
+# define ALT_LAYOUT     _MBL // Layout to display 'MR' led when active.
+#endif
+
+static bool win_key_locked = false;
+static bool mac_media_locked = false;
+static bool win_appkey_pressed = false;
+static bool mac_fnkey_pressed = false;
+
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    KC_TGUI = SAFE_RANGE,  // Toggle between GUI Lock or Unlock (Windows)
+    KC_TMED,               // Toggle between Fx keys or Media keys (Mac)
+    KC_WFN,                // Windows function key
+    KC_MFN,                // Mac function key
+    MO_WSL,                // Windows system key
+    MO_MSL,                // Mac system key
+    DF_W2MBL,              // Change layout from Windows to Mac
+    DF_M2WBL               // Change layout from Mac to Windows
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Keymap _WBL: Windows Base Layer (Default Layer)
+   * ┌───┐   ┌───┬───┬───┬───┐ ┌───┬───┬───┬───┐ ┌───┬───┬───┬───┐ ┌───┬───┬───┐
+   * │Esc│   │F1 │F2 │F3 │F4 │ │F5 │F6 │F7 │F8 │ │F9 │F10│F11│F12│ │PSc│Slk│Pse│
```

Comment:
Don't use NBSPs for spacing.

Reply:
I'm not sure what you mean, could you elaborate a bit?

Reply:
These ASCII art comments use NBSP characters rather than normal spaces. For whatever reason a lot of text editors don't classify them as whitepace characters, and the result is that the ASCII art looks slightly better even when you have the "show whitepace" setting turned on:
![image](https://user-images.githubusercontent.com/4781841/124751827-6d682000-df6a-11eb-82db-a2574d1ea27e.png)
However, it seems this is not universal anymore - Sublime Text 4 now renders NBSPs as `<0xa0>`... which messes up all the nice formatting.
![image](https://user-images.githubusercontent.com/4781841/124752048-b0c28e80-df6a-11eb-9edb-9810469196ec.png)

Reply:
Gotcha, thanks for catching that